huxley/core/admin/delegate.py

```diff
@@ -29,6 +30,29 @@ def roster(self, request):
 
         return roster
 
+    def load(self, request):
+        '''Loads new Assignments.'''
+        delegates = request.FILES
+        reader = csv.reader(delegates['csv'])
+
+        assignments = {}
+        for assignment in Assignment.objects.all():
+            assignments[assignment.committee.name.encode('ascii', 'ignore'),
+             assignment.country.name.encode('ascii', 'ignore'),
+             assignment.school.name, 
+             ] = assignment
+        for row in reader:
+            if row[1] == 'Committee':
+                continue
+            assignment = assignments[unicode(row[1], errors='ignore'),
+                                     unicode(row[2], errors='ignore'),
+                                     row[3], 
+                                    ]
+            d = Delegate.objects.create(name=row[0], assignment=assignment)
+            d.save()
```

Comment:
`create` is a shortcut for constructing and saving a model. This is a redundant call.